ivy/functional/backends/paddle/data_type.py

```diff
@@ -156,15 +173,7 @@ def broadcast_to(
     if x.ndim > len(shape):
         x = x.reshape([-1])
 
-    if x.dtype in [
```

Comment:
In this case, given that the number of unsupported dtypes is lesser than the number of supported ones, we can just specify the unsupported ones using `with_unsupported_dtypes`. Same applies to the other similar instances in the `paddle` backend